content/en/security_platform/cspm/findings.md

```diff
@@ -28,27 +28,33 @@ Cloud Security Posture Management is not currently available in US1-FED or US3.
 
 The Findings page helps drill into the details of CSPM findings, the configuration of resources, and descriptions and remediation guidelines for remediating a misconfigured resource.
```

Comment:
The Findings Explorer is dedicated to exploring the details of your security posture, including the configuration of individual resources and how they conform to [default rules][1], at any point in the past.